db/drivers/mysql/grass-mysql.html

```diff
@@ -29,12 +29,13 @@ <h2>Driver and database name</h2>
 <ul>
     <li> Database name - in case of connection from localhost</li>
     <li> String of comma separated list of kye=value options.
```

Comment:
Can you fix the typo of `key` at the same time?

Reply:
Yes I can. Fixed with https://github.com/OSGeo/grass/pull/4778/commits/936081d01157aef218bc8e4395d9221159a3da4c.